### DISCUSSION TITLE
French translation

### DISCUSSION THREAD

**Discussion by xerta555** [community/community#37999] [Mobile] [answered]
Hi

As i see, the Github app (even the Beta version) isn't translated in French language.

I can do that, but i don't have found where i could purpose it.

Have a good day.

**Comment by stevepopovich** [comment, marked as answer]
Hi Rom, yes you are right, we don't support French at this time. I will file this as an issue with the team to triage. Thanks!

**Reply by xerta555** [reply]
> Hi Rom, yes you are right, we don't support French at this time. I will file this as an issue with the team to triage. Thanks!

From what i understand, it's not possible to offer a direct translation with a file base ?

**Reply by stevepopovich** [reply]
Well, we wouldn't translate code or files, but we would translate all the strings in our app.

**Reply by xerta555** [reply]
> Well, we wouldn't translate code or files, but we would translate all the strings in our app.

I have already translated more than ten applications, i know how to translate Android applications, my question was: can i participate to accelerate the deployment in Beta?

**Reply by stevepopovich** [reply]
oh, I see. Well, our app is closed source right now so we can't accept help from people outside the GitHub organization. We will keep this in mind though, thanks!

**Reply by PPORCH3bis** [reply]
The app still doesn't have French in 1 year. So how can I/we help you to translate it into French ?

**Reply by xerta555** [reply]
@stevepopovich Hi, is the French translation at least planned?

**Reply by stevepopovich** [reply]
Hi, at this point we do not have French planned. We have added other languages in the last few months and improved the quality of them. This issue is in our backlog and we will consider it when we look at adding other languages. Thanks for the interest!
